lib/utils.js

```diff
@@ -135,17 +145,29 @@ const utils = (module.exports = {
           );
         }
 
-        if (params.api_key) {
-          opts.auth = params.api_key;
+        /**
+         * TODO: Remove snake case params in next major version
+         */
+        if (params.api_key || params.apiKey) {
+          opts.auth = params.apiKey || params.api_key;
```

Comment:
Why are the assignments in the opposite order from the `if()`?

Reply:
Oops my bad, fixed now.